tests/testlib/s2n_testlib.h

```diff
@@ -194,8 +196,8 @@ S2N_RESULT s2n_connection_set_test_master_secret(struct s2n_connection *conn, co
 
 #define S2N_TEST_TRUST_STORE "../pems/trust-store/ca-bundle.crt"
 
-#define S2N_DEFAULT_TEST_CERT_CHAIN  S2N_RSA_2048_PKCS1_CERT_CHAIN
```

Comment:
Are there any tests that use `S2N_RSA_2048_PKCS1_CERT_CHAIN` directly rather than via `S2N_DEFAULT_TEST_CERT_CHAIN`? If so, do we expect these tests to fail on RHEL9 as well? If not, can we just remove these pems or replace them with the new ones?

Reply:
There are ones that explicitly test use of SHA1. For example:
https://github.com/aws/s2n-tls/blob/7ec1446ac1d836cefb6a5363b9bfb438a4faa792/tests/unit/s2n_x509_validator_certificate_signatures_test.c#L90-L114

https://github.com/aws/s2n-tls/blob/7ec1446ac1d836cefb6a5363b9bfb438a4faa792/tests/unit/s2n_x509_validator_certificate_signatures_test.c#L164-L216

And there are more so we should probably still keep the old cert chain.

Above tests did not show up on the list of errors in user's issue but this following test does and could be a problem, since it happens at a later point in test and uses `S2N_RSA_2048_PKCS1_CERT_CHAIN` directly.

https://github.com/aws/s2n-tls/blob/7ec1446ac1d836cefb6a5363b9bfb438a4faa792/tests/unit/s2n_x509_validator_test.c#L1894-L1978

However these shouldn't cause errors because the first test is turning off cert validation, and second test is using TLS1.3, which does not accept SHA1.

Of all the errors user listed, which are 

	 17 - s2n_cert_validation_callback_test (Failed)
	 55 - s2n_config_test (Failed)
	 63 - s2n_crl_test (Failed)
	130 - s2n_mutual_auth_test (Failed)
	262 - s2n_x509_validator_test (Failed)
	263 - s2n_x509_validator_time_verification_test (Failed)

only `s2n_x509_validator_test` explicitly uses `S2N_RSA_2048_PKCS1_CERT_CHAIN` so there shouldn't be problem elsewhere